aspnetcore/fundamentals/openapi/include-metadata.md

```diff
@@ -504,9 +504,28 @@ Properties can also be marked as `required` with the [required](/dotnet/csharp/l
 #### enum
 
 Enum types in C# are integer-based, but can be represented as strings in JSON with a  [`[JsonConverter]`](xref:System.Text.Json.Serialization.JsonConverterAttribute) and a <xref:System.Text.Json.Serialization.JsonStringEnumConverter>. When an enum type is represented as a string in JSON, the generated schema will have an `enum` property with the string values of the enum.
+
+The following example demonstrates how to use a `JsonStringEnumConverter` to represent an enum as a string in JSON:
+
+```csharp
+[JsonConverter(typeof(JsonStringEnumConverter<DayOfTheWeekAsString>))]
+public enum DayOfTheWeekAsString
+{
+    Sunday,
+    Monday,
+    Tuesday,
+    Wednesday,
+    Thursday,
+    Friday,
+    Saturday
+}
+```
+
+A special case is when an enum type has the [Flags] attribute, which indicates that the enum can be treated as a bit field; that is, a set of flags. A flags enums with a [JsonConverterAttribute] will be defined as `type: string` in the generated schema with no `enum` property, since the value could be any combination of the enum values.
```

Comment:
Fix typo:
```suggestion
A special case is when an enum type has the [Flags] attribute, which indicates that the enum can be treated as a bit field; that is, a set of flags. A flags enum with a [JsonConverterAttribute] will be defined as `type: string` in the generated schema with no `enum` property, since the value could be any combination of the enum values.
```